test/integration/db/db_tests.dart

```diff
@@ -458,183 +458,186 @@ runTests(db.DatastoreDB store, String namespace) {
+              },
+              // Expando queries: Filter on expanded String property
+                  () async {
+                var query = store.query<ExpandoPerson>(partition: partition)
                   ..filter('foo =', (expandoPersons.last as dynamic).foo)
                   ..run();
-              var models = await runQueryWithExponentialBackoff(query, 1);
-              compareModels([expandoPersons.last], models);
-            },
-            // Expando queries: Filter on expanded int property
-            () async {
-              var query = store.query<ExpandoPerson>(partition: partition)
+                var models = await runQueryWithExponentialBackoff(query, 1);
+                compareModels([expandoPersons.last], models);
+              },
+              // Expando queries: Filter on expanded int property
+                  () async {
+                var query = store.query<ExpandoPerson>(partition: partition)
                   ..filter('bar =', (expandoPersons.last as dynamic).bar)
                   ..run();
-              var models = await runQueryWithExponentialBackoff(query, 1);
-              compareModels([expandoPersons.last], models);
-            },
-            // Expando queries: Filter normal property with different
-            // propertyName (datastore name is 'NN').
-            () async {
-              var query = store.query<ExpandoPerson>(partition: partition)
+                var models = await runQueryWithExponentialBackoff(query, 1);
+                compareModels([expandoPersons.last], models);
+              },
+              // Expando queries: Filter normal property with different
+              // propertyName (datastore name is 'NN').
+                  () async {
+                var query = store.query<ExpandoPerson>(partition: partition)
                   ..filter('nickname =', expandoPersons.last.nickname)
                   ..run();
-              var models = await runQueryWithExponentialBackoff(query, 1);
-              compareModels([expandoPersons.last], models);
-            },
-
-            // Delete results
-            () => store.commit(deletes: allKeys),
-
-            // Wait until the entity deletes are reflected in the indices.
-            () => waitUntilEntitiesGone(store, allKeys, partition),
-
-            // Make sure queries don't return results
-            () => store.lookup(allKeys).then((List<db.Model> models) {
-              expect(models.length, equals(allKeys.length));
-              for (var model in models) {
-                expect(model, isNull);
-              }
-            }),
-          ];
-          return Future.forEach(tests, (f) => f());
+                var models = await runQueryWithExponentialBackoff(query, 1);
+                compareModels([expandoPersons.last], models);
+              },
+
+              // Delete results
+                  () => store.commit(deletes: allKeys),
```

Comment:
Does `dartfmt` produce this indentation? If so, maybe we should file a bug with dartfmt/dart_style?

Reply:
IntelliJ produces it when block reformatting (did that to add/remove brackets).   I haven't run dartfmt yet because the diffs are pretty large for the whole repository and didn't want to conflate those. 

Still though, this is so bad that I manually adjusted it in this section and will land a dartfmt fix for the whole repo later.